<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Exercise</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineSearchExercises">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Search Exercise</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listExercises"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnDeleteExercise">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnEditExercise">
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAddExercise">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Workout Plans</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="lineSearchPlans">
          <property name="placeholderText">
           <string>Search Plans</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listPlans"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnAddPlan">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnEditPlan">
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnViewPlan">
          <property name="text">
           <string>View</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDeletePlan">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
